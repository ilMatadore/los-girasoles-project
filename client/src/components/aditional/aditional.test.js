import React from 'react';
import { shallow } from 'enzyme';
import Aditional from './aditional.component';

let wrapper;

beforeEach(() => {
    const mockProps = {
        adicional: {
            name: 'Mizuna',
            price: 20,
            id: 1,
        },
        addItem: jest.fn()         
    }
    wrapper =  shallow(<Aditional {...mockProps}/>)
})

it('Expect to render Aditional component', () => {
    expect(wrapper).toMatchSnapshot();   
})

it('Test Button', () => {
    wrapper.find(".adicional-cart-icon").simulate('click')
    expect(wrapper.length).toEqual(1)
})