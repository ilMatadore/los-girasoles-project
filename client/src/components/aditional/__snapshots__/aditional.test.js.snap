// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Expect to render Aditional component 1`] = `
<div
  className="adicional"
  data-aos="flip-down"
>
  <div
    className="adicional-title"
  >
    <p
      style={
        Object {
          "paddingRight": "10px",
        }
      }
    >
      Mizuna
    </p>
  </div>
  <div
    style={
      Object {
        "alignItems": "center",
        "display": "flex",
        "float": "right",
        "justifyContent": "flex-end",
        "width": "30%",
      }
    }
  >
    <span
      className="adicional-price"
    >
      $
      20
    </span>
    <button
      className="adicional-button"
    >
      <FaCartPlus
        className="adicional-cart-icon"
        onClick={[Function]}
      />
    </button>
  </div>
</div>
`;
